client/my-sites/stats/features/modules/stats-locations/stats-locations.tsx

```diff
@@ -131,6 +131,8 @@ const StatsLocations: React.FC< StatsModuleLocationsProps > = ( { query, summary
 		/>
 	);
 
+	const hasLocationData = Array.isArray( data ) && data.length;
```

Comment:
```suggestion
	const hasLocationData = Array.isArray( data ) && data.length > 0;
```

Let's be explicit :-)

Reply:
Thanks!